legal-api/src/legal_api/services/filings/validations/common_validations.py

```diff
@@ -88,3 +91,43 @@ def validate_shares(item, memoize_names, filing_type, index) -> Error:
         msg.extend(series_msg)
 
     return msg
+
+
+def validate_court_order(court_order_path, court_order):
+    """Validate the courtOrder data of the filing."""
+    msg = []
+
+    # TODO remove it when the issue with schema validation is fixed
```

Comment:
There is a bug that is not validating the court order.  Maybe because courtOrder is optional but it has required fields... I'm going to create a ticket to Thor as discussed with him.